src/hpke/registry.rs

```diff
@@ -0,0 +1,53 @@
+use super::{IpaPrivateKey, IpaPublicKey, KeyIdentifier};
+
+/// A pair of secret key and public key. Public keys used by UA to encrypt the data towards helpers
+/// secret keys used by helpers to open the ciphertexts. Each helper needs access to both
+/// TODO: we may decide to use different HPKE settings for each key identifier.
```

Comment:
That's a big lift.  It's a classic failing in software to not prepare for the obsolescence of the crypto you are using.  And rust only makes it works.

On the other hand, I don't see a way to fix that without making the code much worse, so let's not concern ourselves with it.

```suggestion
```